### PR TITLE
Fix to compile / configuration errors

### PR DESCRIPTION
**Pull request by gerring**
These were seen with e4 oxygen and Java9 based on the new compliance
settings set in scanning.

### CODE REVIEW

File: org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenPerspective.java [hunk 1]
```diff
@@ -11,15 +11,16 @@
...
-import org.eclipse.ui.IFolderLayout;
 
 public class XcenPerspective implements IPerspectiveFactory {
 
 	/**
 	 * Creates the initial layout for a page.
 	 */
+	@Override
 	public void createInitialLayout(IPageLayout layout) {
 		String editorArea = layout.getEditorArea();
```
Comment (gerring) [inline org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenPerspective.java]: MAJOR Remove this useless assignment to local variable "editorArea". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
MINOR Remove this unused "editorArea" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)

**Comment by gerring** [comment]
SonarQube analysis reported 22 issues
* ![CRITICAL][CRITICAL] 5 critical
* ![MAJOR][MAJOR] 8 major
* ![MINOR][MINOR] 7 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [XcenPerspective.java#L50](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenPerspective.java#L50): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [XcenPerspective.java#L56](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenPerspective.java#L56): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [XcenPerspective.java#L62](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenPerspective.java#L62): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [XcenServices.java#L36](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L36): Make the enclosing method "static" or remove this set. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [XcenServices.java#L40](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L40): Make the enclosing method "static" or remove this set. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)
1. ![MAJOR][MAJOR] [XcenPerspective.java#L50](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenPerspective.java#L50): Remove this unused method parameter "layout". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)
1. ![MAJOR][MAJOR] [XcenPerspective.java#L56](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenPerspective.java#L56): Remove this unused method parameter "layout". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)
1. ![MAJOR][MAJOR] [XcenPerspective.java#L62](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenPerspective.java#L62): Remove this unused method parameter "layout". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)
1. ![MAJOR][MAJOR] [XcenServices.java#L35](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L35): Remove this unused method parameter "context". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)
1. ![MAJOR][MAJOR] [XcenView.java#L271](https://github.com/eclipse/scanning/blob/e24f2ae7f4a6c97d2f545b1e4d552cb3c1c2f9d1/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L271): Define and throw a dedicated exception instead of using a generic one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Mark-Booth** [comment]
http://jenkins.diamond.ac.uk:8080/job/GDA.master-junit.tests-gerrit/7612/

**Comment by Mark-Booth** [comment]
Gerrit job succeeded.

**Comment by gerring** [comment]
Good that gerrit job worked, I guess that does not have sonarqube?

**Comment by gerring** [comment]
SonarQube analysis reported 14 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 6 minor
* ![INFO][INFO] 2 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [XcenServices.java#L36](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L36): Make the enclosing method "static" or remove this set. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [XcenServices.java#L40](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L40): Make the enclosing method "static" or remove this set. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)
1. ![MAJOR][MAJOR] [XcenServices.java#L35](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L35): Remove this unused method parameter "context". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)
1. ![MAJOR][MAJOR] [XcenView.java#L271](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L271): Define and throw a dedicated exception instead of using a generic one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
1. ![MAJOR][MAJOR] [XcenView.java#L332](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L332): Remove this useless assignment to local variable "toolbarManager". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [XcenView.java#L340](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L340): Remove this useless assignment to local variable "menuManager". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MINOR][MINOR] [XcenServices.java#L70](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L70): Reorder the modifiers to comply with the Java Language Specification. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
1. ![MINOR][MINOR] [XcenView.java#L212](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L212): Remove this unnecessary cast to "IToolPageSystem". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)
1. ![MINOR][MINOR] [XcenView.java#L223](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L223): Immediately return this expression instead of assigning it to the temporary variable "system". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)
1. ![MINOR][MINOR] [XcenView.java#L240](https://github.com/eclipse/scanning/blob/e317f6f1d5d909543396911249c31fb0c8bc7775/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L240): Use isEmpty() to check whether the collection is empty or not. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1155)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java [hunk 2]
```diff
@@ -35,26 +32,29 @@ public void setLoaderService(ILoaderService loaderService) {
 
 	private static XcenServices current;
 
-	public void start(ComponentContext context) {
-		current = this;
+	public void  start(ComponentContext context) {
```
Comment (gerring) [inline org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java]: MAJOR Remove this unused method parameter "context". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)

File: org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java [hunk 3]
```diff
@@ -35,26 +32,29 @@ public void setLoaderService(ILoaderService loaderService) {
...
+		XcenServices.current = this;
```
Comment (gerring) [inline org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java]: CRITICAL Make the enclosing method "static" or remove this set. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)

File: org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java [hunk 4]
```diff
@@ -35,26 +32,29 @@ public void setLoaderService(ILoaderService loaderService) {
...
 	}
 
 	public void stop() {
-		current = null;
+		XcenServices.current = null;
```
Comment (gerring) [inline org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java]: CRITICAL Make the enclosing method "static" or remove this set. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)

**Comment by gerring** [comment]
SonarQube analysis reported 14 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 6 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [XcenView.java#L271](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L271): Define and throw a dedicated exception instead of using a generic one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
1. ![MAJOR][MAJOR] [XcenView.java#L332](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L332): Remove this useless assignment to local variable "toolbarManager". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [XcenView.java#L340](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L340): Remove this useless assignment to local variable "menuManager". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MINOR][MINOR] [XcenServices.java#L70](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L70): Reorder the modifiers to comply with the Java Language Specification. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
1. ![MINOR][MINOR] [XcenView.java#L212](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L212): Remove this unnecessary cast to "IToolPageSystem". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)
1. ![MINOR][MINOR] [XcenView.java#L223](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L223): Immediately return this expression instead of assigning it to the temporary variable "system". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)
1. ![MINOR][MINOR] [XcenView.java#L240](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L240): Use isEmpty() to check whether the collection is empty or not. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1155)
1. ![MINOR][MINOR] [XcenView.java#L332](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L332): Remove this unused "toolbarManager" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)
1. ![MINOR][MINOR] [XcenView.java#L340](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L340): Remove this unused "menuManager" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)
1. ![INFO][INFO] [XcenView.java#L149](https://github.com/eclipse/scanning/blob/d03e7e2b277e88940bdf08b2050d4bdf235d9629/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L149): Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java [hunk 5]
```diff
@@ -36,25 +33,32 @@ public void setLoaderService(ILoaderService loaderService) {
 	private static XcenServices current;
 
 	public void start(ComponentContext context) {
```
Comment (gerring) [inline org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java]: MAJOR Remove this unused method parameter "context". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)

**Comment by gerring** [comment]
SonarQube analysis reported 12 issues
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 6 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [XcenView.java#L271](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L271): Define and throw a dedicated exception instead of using a generic one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
1. ![MAJOR][MAJOR] [XcenView.java#L332](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L332): Remove this useless assignment to local variable "toolbarManager". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [XcenView.java#L340](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L340): Remove this useless assignment to local variable "menuManager". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MINOR][MINOR] [XcenServices.java#L74](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java#L74): Reorder the modifiers to comply with the Java Language Specification. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
1. ![MINOR][MINOR] [XcenView.java#L212](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L212): Remove this unnecessary cast to "IToolPageSystem". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)
1. ![MINOR][MINOR] [XcenView.java#L223](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L223): Immediately return this expression instead of assigning it to the temporary variable "system". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)
1. ![MINOR][MINOR] [XcenView.java#L240](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L240): Use isEmpty() to check whether the collection is empty or not. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1155)
1. ![MINOR][MINOR] [XcenView.java#L332](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L332): Remove this unused "toolbarManager" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)
1. ![MINOR][MINOR] [XcenView.java#L340](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L340): Remove this unused "menuManager" local variable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)
1. ![INFO][INFO] [XcenView.java#L149](https://github.com/eclipse/scanning/blob/3511d607acd8f25844ab51fb9694d83d69796332/org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/views/XcenView.java#L149): Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Mark-Booth** [comment]
If you want someone to run gerrit integration tests before you merge in a change @gerring just as me as a reviewer and I'll do it for you.

**Pull request merged**